node/subsystem/src/messages.rs

```diff
@@ -290,7 +290,7 @@ pub struct SchedulerRoster {
 
 /// A description of an error causing the runtime API request to be unservable.
 #[derive(Debug, Clone)]
-pub struct RuntimeApiError(String);
+pub struct RuntimeApiError(pub String);
```

Comment:
not a fan of leaking implementation details in the public API, we probably only need `From<&str>` and `Display` impl as an interface